gui/src/app/direct/directHeader.tpl.html

```diff
@@ -35,12 +35,14 @@
             <li uib-dropdown ng-class="{active: $state.includes('direct.ccda') || $state.includes('direct.ccdar2') || $state.includes('direct.ccdar3')}">

```

Comment:
Should there maybe be:
 || $state.includes('direct.ccdasvap2022')
...or something like that? And, is there a CSS class created? And, if not, maybe we don't the ng-class for the other items at all, or those CSS classes?